tests/components/knx/test_config_flow.py

```diff
@@ -510,7 +510,7 @@ async def test_routing_secure_keyfile(
     return_value=GatewayScannerMock(),
 )
 async def test_tunneling_setup_manual(
-    _gateway_scanner_mock,
+    gateway_scanner_mock: MagicMock,
```

Comment:
wouldn't the idea of https://docs.astral.sh/ruff/rules/pytest-fixture-param-without-value/ be to use
```py
@pytest.mark.usefixtures("gateway_scanner_mock")
async def test_tunneling_setup_manual(
```
instead?

Reply:
See https://github.com/astral-sh/ruff/issues/11463

Replacing the patch with a fixture is possible - but then it become much more verbose as you first need to create the fixtures.

Reply:
Just for my understanding: we remove the `_` prefix to avoid a bug in the ruff checker? 

This seems counterintuitive to me, but if you like it more that way, I have no strong feelings about test linting 🙃

Reply:
I originally thought to use a `noqa` statement, but I think in this case renaming the variable is the cleanest solution.